### DISCUSSION TITLE
User Guide and Definitions

### DISCUSSION THREAD

**Discussion by nodecentral** [wger-project/wger#1233] [Q&A]
Hi, 

Is there an end user guide/document available that clearly explains what all the menu options are ? 

Even something as key in Wger as ‘Workout’ , what exactly does that mean, and once created how best do I define the training day etc.. 

For me, while I go to the gym quite frequently, at times there is no plan, I just enjoy the various equipment that it has etc. Now, for that latter point, Wger does not seem to accommodate that very well,, as everything needs to be planned in advance.. 

It would be really helpful to see a real world example in the system, for example a 3 day workout plan where on 1 day, you do  Chest, and Triceps, another day it’s Back and Biceps and then on the final day it’s Legs and Shoulders..

**Comment by rolandgeider** [comment]
There is an issue already somewhere to make the terminology more clear, so e.g. "workout" would become a "workout routine" because it's more of a plan and the logs would be a specific workout you did on one specific day. The backend already supports saving any exercises so it's not planned, we just need to make the frontent and the app aware of that (and, you guessed it, there's also a ticket for that as well)

**Comment by nodecentral** [comment]
Thanks for that @rolandgeider..

I must admit, I’ve been struggling with how to use the ‘Workout’ section, so much so that i’ve initially been using it as a way to plan my ‘gym visits’, but similar to if I use it for ‘routines’, why does it have an ability to choose multiple days. Much like like a ‘trip to the gym’ how can a ‘routine’ span different days?…

Either-way, based on what you’re saying - “A Workout routine is a group of exercises, that are to be done either on one day, or over multi days”

I’m still not sure how best to use this, especially as my routines can vary a bit, so a ‘leg day’ may not follow the exact same exercises every time, And if it ties into the calendar too, that can confuse things, when i need to log things..

On that later point, Is there an ability somewhere in Wger for me to just log the exercises I do (record the type and associated number of sets/reps) , but it not require/be part of an overall ‘Workout Routine’ ?

**Comment by laur89** [comment]
@rolandgeider I somewhat agree with OP. Perhaps a real-world use-case scenario would offer the best ROI? To show how one, from practical point of view, might use it to plan/schedule/track their next week/2weeks of workouts.

Related: https://github.com/wger-project/wger/discussions/1271
